src/Microsoft.Health.Dicom.Api/Controllers/WorkitemController.Add.cs

```diff
@@ -46,23 +49,23 @@ public partial class WorkitemController
     [VersionedPartitionRoute(KnownRoutes.AddWorkitemInstancesRoute, Name = KnownRouteNames.PartitionedAddWorkitemInstance)]
     [VersionedRoute(KnownRoutes.AddWorkitemInstancesRoute, Name = KnownRouteNames.AddWorkitemInstance)]
     [AuditEventType(AuditEventSubType.AddWorkitem)]
-    public async Task<IActionResult> AddAsync()
+    public async Task<IActionResult> AddAsync([FromBody][Required][MinLength(1)][MaxLength(1)] IReadOnlyList<DicomDataset> dicomDatasets)
```

Comment:
I think the only thing we want to check before merging this is how we expose these errors as per the UPS-RS standard

Reply:
QueryStringValidatorMiddleware kicks in and throws 400 error with message "The query parameter '' is invalid. A non-empty request body is required.".

![image](https://user-images.githubusercontent.com/94023192/184365420-d27229f4-c46c-4a81-bc67-ec282a1350df.png)

Reply:
Why does it talk about query parameter and also body?

Reply:
It is due to [Required] and [MinLength] attributes.